spec/controllers/post_controller_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
+  describe '#index' do
+    before do
```

Comment:
don't forget you can replace this with a simple `let` statement
 Like: `let(:user) { FactoryBot.create(:user) }`

[OPTIONAL] Try doing it for post specs as well

Reply:
But you said to use `before` in [this comment](https://github.com/bolah2009/facebook-clone/pull/3#discussion_r312508034) :disappointed:

Reply:
It's about context of the situation. In this case, unless you have create a new user for each test, declaring it in let statement makes most sense.

However, in other comment, you are signing in, which requires you sign in each time. Does this make sense 😅